core/workspace_audio.js

```diff
@@ -98,10 +99,10 @@ Blockly.WorkspaceAudio.prototype.load = function(filenames, name) {
  * @package
  */
 Blockly.WorkspaceAudio.prototype.preload = function() {
-  for (var name in this.SOUNDS_) {
-    var sound = this.SOUNDS_[name];
+  for (let name in this.SOUNDS_) {
```

Comment:
I think we decided to have these be `const`, or is that only for `for ... of` loops?

Reply:
Updated.